blood.go

```diff
@@ -0,0 +1,16 @@
+package faker
+
+// Gender is a faker struct for Gender
+type Blood struct {
+	Faker *Faker
+}
+
+// Name returns a Gender name for Gender
+func (f Blood) Name() string {
```

Comment:
What is the difference between `Name` and `Abbr`? They are returning random values from the same list.